protocol/x/ratelimit/client/cli/query_capacity_by_denom_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build all || integration_test
+
+package cli_test
+
+import (
+	"fmt"
+	"math/big"
+	"strconv"
+	"testing"
+
+	tmcli "github.com/cometbft/cometbft/libs/cli"
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+	"github.com/dydxprotocol/v4-chain/protocol/dtypes"
+	assettypes "github.com/dydxprotocol/v4-chain/protocol/x/assets/types"
+	"github.com/dydxprotocol/v4-chain/protocol/x/ratelimit/client/cli"
+	"github.com/dydxprotocol/v4-chain/protocol/x/ratelimit/types"
+	ratelimitutil "github.com/dydxprotocol/v4-chain/protocol/x/ratelimit/util"
+	"github.com/stretchr/testify/require"
+)
+
+// Prevent strconv unused error
+var _ = strconv.IntSize
+
+func TestQueryCapacityByDenom(t *testing.T) {
+	net, ctx := setupNetwork(t)
+
+	out, err := clitestutil.ExecTestCLICmd(ctx,
+		cli.CmdQueryCapacityByDenom(),
+		[]string{
+			fmt.Sprintf("--%s=json", tmcli.OutputFlag),
+			assettypes.AssetUsdc.Denom,
+		})
+
+	require.NoError(t, err)
+	var resp types.QueryCapacityByDenomResponse
+	require.NoError(t, net.Config.Codec.UnmarshalJSON(out.Bytes(), &resp))
+	require.Equal(t,
+		[]types.LimiterCapacity{
```

Comment:
For my understanding, is this expected list of `LimiterCapacity` from default genesis?

Reply:
Yes, added comment to clarify.